packages/webchat/src/declaration.d.ts

```diff
@@ -0,0 +1 @@
+declare module '*.scss'
```

Comment:
There's already this line in globals.d.ts. Why do we need this file also?

Reply:
I moved this line into this file because it was clashing with the other module I declare in the global typings.